test/fixtures/keys/Makefile

```diff
@@ -24,13 +24,37 @@ all: \
   dh512.pem \
   dh1024.pem \
   dh2048.pem \
+	dherror.pem \
+	dsa_params.pem \
+	dsa_private.pem \
+	dsa_private_encrypted.pem \
+	dsa_private_pkcs8.pem \
+	dsa_public.pem \
   dsa1025.pem \
   dsa_private_1025.pem \
   dsa_private_encrypted_1025.pem \
   dsa_public_1025.pem \
   ec-cert.pem \
   ec.pfx \
   fake-cnnic-root-cert.pem \
+	rsa_private.pem \
```

Comment:
This list seems to be mixing tabs and double spaces? (There are tabs in other parts of this file too, but I guess those are intended)

Reply:
I've changed the indenting to match the rest of the file.